PowerPointLabs/PowerPointLabs/TextCollection/SaveLabText.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace PowerPointLabs.TextCollection
+{
+    internal static class SaveLabText
+    {
+        #region Action Framework Tags
+        public const string RibbonMenuId = "SaveLabMenu";
+        public const string SavePresentationsButtonTag = "SavePresentations";
+        public const string SaveLabSettingsButtonTag = "SaveLabSettings";
+        #endregion
+
+        #region GUI Text
+        public const string RibbonMenuLabel = "Save";
+        public const string SavePresentationsButtonLabel = "Save Presentations";
```

Comment:
Should this be "Save slides" or "Export slides" instead?

Reply:
Lets remain as "Save Presentation" for greater clarity as it belongs to Save Lab

Reply:
Ok then maybe shouldn't be plural form. Yeah "Save Presentation" sounds okay...